spec/unit/embedded.spec.ts

```diff
@@ -668,10 +680,106 @@ describe("RoomWidgetClient", () => {
                     detail: { data: { state: [event] } },
                 }),
             );
+            // Allow the getClientVersions promise to resolve
+            await new Promise<void>((resolve) => setTimeout(resolve, 0));
```

Comment:
Can we use mock `getClientVersions` or jest fake timers instead ?   (`flushPromises` if we can't)